### PR TITLE
tp-qemu: correct block_stream_simple doc string

### PR DESCRIPTION
**Pull request by ShaolongHu**
Correct block_stream_simple doc string for speed related instruction.

Signed-off-by: Shaolong Hu <shu@redhat.com>

### CODE REVIEW

**Comment by ShaolongHu** [comment]
ID: 1232682

File: qemu/tests/block_stream_simple.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def query_status(self):
 def run(test, params, env):
     """
     block_stream_simple test:
-    1). launch block streaming job w/o set max speed
+    1). launch block streaming job w/ max speed in param "default_speed"
```
Comment (xutian) [inline qemu/tests/block_stream_simple.py]: default configure file has default_speed setting but it's configurable for user, if user drop default_speed param  by line "def default_speed",  block stream job will start by monitor command  without  speed param else block stream job will start by monitor command with speed param, so should say "with or without default speed" looks more reasonable.

**Comment by ShaolongHu** [comment]
Have updated. @xutian

**Comment by xutian** [comment]
Acked-by:  Xu Tian <xutian@redhat.com>

**Comment by spcui** [comment]
LGTM
Acked-by: @spcui

**Pull request merged**